crates/ruff_linter/src/codes.rs

```diff
@@ -1081,6 +1081,9 @@ pub fn code_to_rule(linter: Linter, code: &str) -> Option<(RuleGroup, Rule)> {
         (Flake8Logging, "007") => (RuleGroup::Stable, rules::flake8_logging::rules::ExceptionWithoutExcInfo),
         (Flake8Logging, "009") => (RuleGroup::Stable, rules::flake8_logging::rules::UndocumentedWarn),
 
+        // flake8-markupsafe
+        (Flake8MarkupSafe, "001") => (RuleGroup::Preview, rules::flake8_markupsafe::rules::UnsafeMarkupUse),
```

Comment:
I understand that flake8-markup-safe has only a single rule. If that's indeed the case, then I don't think it's worth for us to introduce a new rule group just for that. Instead, we should add it to the `RUF` group and mentioned it in the documentation that this rule is inherited/inspired by `flake8-markup-safe`

Reply:
That's fair, I brought this up in the original issue as well. I'm fine with whatever you decide on.